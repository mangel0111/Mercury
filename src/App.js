import React, { Component } from 'react';
import Header from './components/Header';
import Merchants from './containers/Merchants';


class App extends Component {
	render() {
		return (
			<div>
				<Header />
				<Merchants />
			</div>
		);
	}
}

export default App;
