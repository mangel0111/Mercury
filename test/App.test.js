import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/App';

jest.mock('../src/containers/Merchants', () => 'merchants');
jest.mock('../src/containers/MerchantTable');

test('renders without crashing', () => {
	const rendered = renderer.create(
		<App/>);
	const tree = rendered.toJSON();
	expect(tree).toMatchSnapshot();
});
