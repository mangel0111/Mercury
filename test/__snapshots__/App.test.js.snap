// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <header
    className="sc-bdVaJa jhfvel"
  >
    <div
      className="sc-bwzfXH hWHZbd"
    >
      <div
        className="sc-htpNat cgSBXb"
      >
        <div>
          Mercury
        </div>
        <div
          className="sc-bxivhb fzMRlP"
        >
          Home
        </div>
      </div>
    </div>
  </header>
  <merchants />
</div>
`;
